Dockerfile

```
@@ -1,7 +1,7 @@
 # ================================
 # Build image
 # ================================
-FROM swift:5.10-jammy as build
+FROM swift:6.0-jammy as build
```

```suggestion
FROM swift:6.0-noble AS build
```

fix `FromAsCasing` warning